index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+       <meta charset="UTF-8" />
+       <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+       <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+       <link rel="stylesheet" href="style.css" />
+       <title>My Mobile Portfolio</title>
+    </head>
+    <body>
+        <header>
+                <div class=""></div>
+                <div class=""></div>
```

Comment:
- [ ] As requested by the [previous reviewer](https://github.com/Bantealem/Portfolio-setup-and-mobile-version-skeleton/pull/1#discussion_r912346915), kindly remove these two empty unused divs.